### PR TITLE
Update web viewer links

### PR DESCRIPTION
**Pull request by jprochazk**
### What

Update most web viewer links from `app.rerun.io` to `rerun.io/viewer`.

- Part of https://github.com/rerun-io/landing/issues/730

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/5738/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/5738/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/5738/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5738)
- [Docs preview](https://rerun.io/preview/ea736f87a22392f5007a465f11b1576fe5537ad7/docs) 
- [Examples preview](https://rerun.io/preview/ea736f87a22392f5007a465f11b1576fe5537ad7/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

**Comment by emilk** [comment]
"most"? Which ones aren't updated, and why?

File: .vscode/settings.json [hunk 1]
```diff
@@ -103,5 +103,6 @@
     },
     "[yaml]": {
         "editor.defaultFormatter": "esbenp.prettier-vscode"
-    }
+    },
+    "rust-analyzer.cargo.target": "wasm32-unknown-unknown"
```
Comment (emilk) [inline .vscode/settings.json:107]: rm

File: examples/python/notebook/cube.ipynb [hunk 2]
```diff
@@ -298,7 +298,7 @@
     "\n",
     "bp = rrb.Blueprint(collapse_panels=True)\n",
     "\n",
-    "rec.notebook_show(blueprint=bp)"
+    "rec.notebook_show(blueprint=bp)\n"
```
Comment (emilk) [inline examples/python/notebook/cube.ipynb:301]: A lot of new `\n` in this file O.o

File: scripts/ci/generate_pr_summary.py [hunk 3]
```diff
@@ -47,7 +47,7 @@ def generate_pr_summary(github_token: str, github_repository: str, pr_number: in
         app_blob = viewer_bucket.blob(f"commit/{commit_short}/index.html")
         if app_blob.exists():
             print(f"Found web assets commit: {commit_short}")
-            found["hosted_app"] = f"https://app.rerun.io/commit/{commit_short}"
+            found["hosted_app"] = f"https://rerun.io/viewer/c/{commit_short}"
```
Comment (emilk) [inline scripts/ci/generate_pr_summary.py:50]: `/c/` seems unnecessary short and cryptic imho.

For reference, `GitHub` uses `commit`:

`https://github.com/rerun-io/rerun/commit/c504eba`

**Review by emilk — APPROVED** [review]

**Comment by jprochazk** [comment]
> Which ones aren't updated, and why?

One example is the `example_manifest.json` links. There's a difference between "link to the web viewer" and "link to the website that hosts web viewer assets". The latter is staying as `app.rerun.io`, because we have no way of hosting those assets at `rerun.io/viewer` or similar.

File: examples/python/notebook/cube.ipynb (hunk 2 above)
Comment (jprochazk) [reply examples/python/notebook/cube.ipynb:301]: That's how VSCode formatted it when I saved the file. Seems fine

**Comment by jprochazk** [comment]
Preview: https://landing-git-jan-viewer-migration-rerun.vercel.app/viewer/pr/5738?manifest_url=https%3A%2F%2Fapp.rerun.io%2Fversion%2Fnightly%2Fexamples_manifest.json

**Pull request merged**